partcad/src/partcad/part_factory_obj.py

```
@@ -0,0 +1,152 @@
+    def _should_use_subprocess(self, file_size):
+        """
+        Determine whether to use a subprocess based on inflight counts and file size.
+        """
+        with PartFactoryObj.lock:
+            if (
+                PartFactoryObj.count_inflight_simple < PartFactoryObj.MIN_SIMPLE_INFLIGHT
+                or file_size < PartFactoryObj.MIN_SUBPROCESS_FILE_SIZE
+            ):
+                PartFactoryObj.count_inflight_simple += 1
+                return False
+            else:
```

Ruff (0.8.2)

Unnecessary `else` after `return` statement

Remove unnecessary `else`

(RET505)